omero/conf.py

```diff
@@ -107,20 +107,6 @@
 # Add any paths that contain custom themes here, relative to this directory.
 html_theme_path.extend(['themes'])
 
-# -- Options for LaTeX output --------------------------------------------------
```

Comment:
Ah, when you said remove I assumed delete rather than comment out - I'll fix that now